wiki/Debugging.md

```diff
@@ -9,4 +9,5 @@ Generally, finding the root cause of a bug is a question of strategically checki
 3. Based on the environment you've identified in steps 1 + 2, write a test case which the bug causes to fail (if one doesn't already exist). This gives you an automatic way to check whether or not the bug is fixed.
 4. Then, work towards the solution, until the test case you wrote in step 3 passes.
 
-Unless the bug you are trying to fix is trivial, we strongly recommend creating a debugging doc to organize your work. You can use this debugging doc to write down any pertinent information about the issue and to organize your debugging process, and it will make it much easier to ask for help from others since all the necessary information is there. You can find more guidance [here](https://github.com/oppia/oppia-android/wiki/Get-Help#before-you-ask-a-general-question) on how to write a debugging doc.
\ No newline at end of file
+Unless the bug you are trying to fix is trivial, we recommend creating a debugging doc to organize your work:
```

Comment:
... to organize your work and easily communicate it to other team members. See [Debugging Docs](...) for guidance on how to write a debugging doc.

Reply:
Done.